torch/ao/sparsity/_mappings.py

```diff
@@ -0,0 +1,25 @@
+import torch
+
+def get_sparse_mapping():
+    _sparse_mapping = dict({
+        torch.nn.Linear: torch.ao.nn.sparse.Linear,
+    })
+    return _sparse_mapping
+
+def get_static_sparse_quantized_mapping():
+    _static_sparse_quantized_mapping = dict({
+        # Dense
```

Comment:
Why are the dense mappings present here? Lets make this simpler: We only map from sparse.Linear to the quantized Linear modules. If someone wants to override this, they can.